everviz/plugins/objectives_plot/single_objectives_plot.py

```diff
@@ -39,20 +40,48 @@ def add_webvizstore(self):
 
     @property
     def layout(self):
+        data = get_data(self.csv_file)
+        exclude = ["batch", "accepted"]
+        functions = [name for name in data.columns if name not in exclude]
+        function_dropdown_options = [{"label": i, "value": i} for i in functions]
+        func_elements = [
+            html.Label("Functions to plot:"),
+            dcc.Dropdown(
+                id=self.function_dropdown_id,
+                options=function_dropdown_options,
+                multi=True,
+                value=[functions[0]],
+            ),
+        ]
         return html.Div(
             [
-                dcc.Graph(
-                    id=self.graph_id,
-                    config={
-                        "modeBarButtonsToRemove": ["toImage"],
-                        "displaylogo": False,
+                html.Div(
+                    func_elements,
+                    style={
+                        "width": "29%",
+                        "display": "inline-block",
+                        "vertical-align": "top",
                     },
-                )
-            ],
-            id=self.div_id,
-            style={"width": "69%", "display": "inline-block"},
+                ),
+                html.Div(
+                    [
+                        dcc.Graph(
+                            id=self.graph_id,
+                            config={
+                                "modeBarButtonsToRemove": ["toImage"],
+                                "displaylogo": False,
+                            },
+                        )
+                    ],
+                    id=self.div_id,
+                    style={"width": "69%", "display": "inline-block"},
+                ),
+            ]
         )
 
+    def plot_data(self, data):
```

Comment:
Is this used?

Reply:
It is not used. I wanted to test something with that function and I forgot to delete it. 

I will remove it now!